src/bundle/createSES.js

```diff
@@ -49,6 +50,12 @@ export function createSESWithRealmConstructor(creatorStrings, Realm) {
       }
     });
 
+    if (sloppyGlobals) {
+      throw TypeError(
+        `sloppyGlobals cannot be specified on frozen Realms (like SES)`,
```

Comment:
I would remove the term "Frozen Realms" as it will be obscure. Just refer to "SES" directly. After all, they will see this in a call to a method with "SES" in its name.

But you should make the error message specific to root realms. A compartment is also a realm, so a compartment within a SES root realm is also a SES realm. The current text (especially after removing "Frozen Realms" would suggest that sloppyGlobals cannot be used on a compartment either.

In fact, you may want to phrase it as a suggestion that perhaps they meant (or should have meant) to create a compartment and use sloppyGlobals there.